packages/invoices/src/api.ts

```diff
@@ -1,60 +1,51 @@
-import { ObjectWithStringKeys } from '@openmsupply-client/common/src/types/utility';
 import {
   request,
   gql,
-  batchRequests,
   Name,
   SortBy,
-  ListApi,
   Invoice,
   GraphQLClient,
   getSdk,
-  InvoiceSortFieldInput,
-  InvoiceRow,
   NameSortFieldInput,
+  InvoiceQuery,
```

Comment:
is there a reason that `ListView` has a separate `api.ts` and the `DetailView` is instead using this `api` in the root of invoices? I'm thinking now to have an `api.ts` in each, what do you think?

Reply:
Ah yes that was my intention. Going to jump on shifting the items/customer stuff shortly so will do it there!